### PR TITLE
Install multiple R versions

### PR DESCRIPTION
**Pull request by DamianBudelewski**
Add task to install versions based on the r-versions list parsed in variables.

### CODE REVIEW

File: tasks/install-r.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- name: install | r-version (download)
+  get_url:
+    url: "https://cdn.rstudio.com/r/ubuntu-2004/pkgs/r-{{ item }}_1_amd64.deb"
```
Comment (shmileee) [inline tasks/install-r.yml:4]: `amd64` can be replaced with `{{ rstudio_connect_machine_map[ansible_machine] }}`. And  `2004` can be probably replace with `{{ ansible_lsb.release replace('.','') }}`.

File: tasks/install-r.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+- name: install | r-version (download)
+  get_url:
+    url: "https://cdn.rstudio.com/r/ubuntu-2004/pkgs/r-{{ item }}_1_amd64.deb" 
+    dest: /tmp/r-{{ item }}_1_amd64.deb
```
Comment (shmileee) [inline tasks/install-r.yml:5]: Use `{{ rstudio_connect_downloads_path }}` for temp files.

File: tasks/install-r.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  loop: "{{ r_versions }}"
+  tags:
+    - r-version-install
+
+- name: install | r-version
+  command: gdebi --n /tmp/r-{{ item }}_1_amd64.deb
```
Comment (shmileee) [inline tasks/install-r.yml:11]: Do we need `gdebi` to install `.deb` package? Ansible's builtin apt module can install `.deb` as well: https://github.com/Appsilon/ansible-rstudio-connect/blob/main/tasks/install.yml#L43

**Review by shmileee — CHANGES_REQUESTED** [review]

File: tasks/install-r.yml (hunk 1 above)
Comment (DamianBudelewski) [reply tasks/install-r.yml]: Added.

File: tasks/install-r.yml (hunk 3 above)
Comment (DamianBudelewski) [reply tasks/install-r.yml]: Tried with apt, worked and pushed with new commit.

**Pull request merged**